src/components/Explore/interfaces.ts

```diff
@@ -21,28 +21,21 @@ export interface Series {
   params?: SeriesParams;
 }
 
-/**
- * Indices assigned to each enumerated explore metric needs to match
- * the order of the dropdown items in EXPLORE_METRICS in Explore/utils.ts.
- *
- * So if we change the ordering of items in EXPLORE_METRICS in Explore/utils.ts,
- * we need to update these assigned indices to match.
- */
 export enum ExploreMetric {
-  RATIO_BEDS_WITH_COVID = 0,
-  ADMISSIONS_PER_100K = 1,
-  WEEKLY_DEATHS = 2,
-  WEEKLY_CASES = 3,
-  CASES = 4,
-  DEATHS = 5,
-  HOSPITALIZATIONS = 6,
-  ICU_HOSPITALIZATIONS = 7,
-  VACCINATIONS_FIRST_DOSE = 8,
-  VACCINATIONS_COMPLETED = 9,
-  VACCINATIONS_ADDITIONAL_DOSE = 10,
-  ICU_USED = 11,
-  POSITIVITY_RATE = 12,
-  DAILY_CASES_PER_100K = 13,
+  RATIO_BEDS_WITH_COVID,
+  ADMISSIONS_PER_100K,
+  WEEKLY_DEATHS,
+  WEEKLY_CASES,
+  CASES,
+  DEATHS,
+  HOSPITALIZATIONS,
+  ICU_HOSPITALIZATIONS,
+  VACCINATIONS_FIRST_DOSE,
+  VACCINATIONS_COMPLETED,
+  VACCINATIONS_ADDITIONAL_DOSE,
+  ICU_USED,
+  POSITIVITY_RATE,
+  DAILY_CASES_PER_100K,
```

Comment:
FWIW- To verify my change works I tested by randomly re-ordering these and verified that it had no effect on the trends menu ordering or the functionality of the metrics.